ncc/examples/monogram.py

```diff
@@ -0,0 +1,468 @@
+    "Ч": [0, 0, 0, 17, 17, 17, 30, 16, 16, 16, 0, 0],
+    "Ш": [0, 0, 0, 21, 21, 21, 21, 21, 21, 31, 0, 0],
+    "Щ": [0, 0, 0, 21, 21, 21, 21, 21, 21, 31, 16, 0],
+    "Ъ": [0, 0, 0, 0, 3, 2, 14, 18, 18, 14, 0, 0],
+    "Ы": [0, 0, 0, 0, 17, 17, 19, 21, 21, 19, 0, 0],
+    "Ь": [0, 0, 0, 0, 1, 1, 15, 17, 17, 15, 0, 0],
+    "Э": [0, 0, 0, 14, 17, 16, 28, 16, 17, 14, 0, 0],
+    "Ю": [0, 0, 0, 9, 21, 21, 23, 21, 21, 9, 0, 0],
+    "Я": [0, 0, 0, 0, 30, 17, 17, 30, 17, 17, 0, 0],
+    "а": [0, 0, 0, 0, 0, 14, 16, 30, 17, 30, 0, 0],
+    "б": [0, 0, 0, 30, 1, 13, 19, 17, 17, 14, 0, 0],
+    "в": [0, 0, 0, 0, 0, 15, 17, 15, 17, 15, 0, 0],
+    "г": [0, 0, 0, 0, 0, 31, 1, 1, 1, 1, 0, 0],
+    "д": [0, 0, 0, 0, 0, 12, 10, 10, 10, 31, 17, 0],
+    "е": [0, 0, 0, 0, 0, 14, 17, 31, 1, 14, 0, 0],
+    "ж": [0, 0, 0, 0, 0, 21, 14, 4, 14, 21, 0, 0],
+    "з": [0, 0, 0, 0, 0, 6, 9, 4, 9, 6, 0, 0],
+    "и": [0, 0, 0, 0, 0, 17, 25, 21, 19, 17, 0, 0],
+    "й": [0, 0, 0, 10, 4, 17, 25, 21, 19, 17, 0, 0],
+    "к": [0, 0, 0, 0, 0, 17, 9, 7, 9, 17, 0, 0],
+    "л": [0, 0, 0, 0, 0, 30, 18, 18, 18, 17, 0, 0],
+    "м": [0, 0, 0, 0, 0, 17, 27, 21, 17, 17, 0, 0],
+    "н": [0, 0, 0, 0, 0, 17, 17, 31, 17, 17, 0, 0],
+    "о": [0, 0, 0, 0, 0, 14, 17, 17, 17, 14, 0, 0],
+    "п": [0, 0, 0, 0, 0, 31, 17, 17, 17, 17, 0, 0],
+    "р": [0, 0, 0, 0, 0, 13, 19, 17, 17, 15, 1, 1],
+    "с": [0, 0, 0, 0, 0, 14, 17, 1, 17, 14, 0, 0],
+    "т": [0, 0, 0, 0, 0, 31, 4, 4, 4, 4, 0, 0],
+    "у": [0, 0, 0, 0, 0, 17, 17, 17, 17, 30, 16, 14],
+    "ф": [0, 0, 0, 4, 4, 14, 21, 21, 21, 14, 4, 4],
+    "х": [0, 0, 0, 0, 0, 17, 10, 4, 10, 17, 0, 0],
+    "ц": [0, 0, 0, 0, 0, 9, 9, 9, 9, 31, 16, 0],
+    "ч": [0, 0, 0, 0, 0, 17, 17, 17, 30, 16, 0, 0],
+    "ш": [0, 0, 0, 0, 0, 21, 21, 21, 21, 31, 0, 0],
+    "щ": [0, 0, 0, 0, 0, 21, 21, 21, 21, 31, 16, 0],
+    "ъ": [0, 0, 0, 0, 0, 3, 2, 14, 18, 14, 0, 0],
+    "ы": [0, 0, 0, 0, 0, 17, 17, 19, 21, 19, 0, 0],
+    "ь": [0, 0, 0, 0, 0, 1, 1, 15, 17, 15, 0, 0],
+    "э": [0, 0, 0, 0, 0, 14, 17, 28, 17, 14, 0, 0],
+    "ю": [0, 0, 0, 0, 0, 9, 21, 23, 21, 9, 0, 0],
+    "я": [0, 0, 0, 0, 0, 30, 17, 17, 30, 17, 0, 0],
+    "ё": [0, 0, 0, 10, 0, 14, 17, 31, 1, 14, 0, 0],
+    "—": [0, 0, 0, 0, 0, 0, 31, 0, 0, 0, 0, 0],
+    "’": [0, 0, 0, 8, 4, 0, 0, 0, 0, 0, 0, 0],
+    "…": [0, 0, 0, 0, 0, 0, 0, 0, 21, 21, 0, 0],
+    "€": [0, 0, 0, 12, 18, 7, 2, 7, 18, 12, 0, 0],
+    "←": [0, 0, 0, 0, 0, 4, 30, 31, 30, 4, 0, 0],
+    "↑": [0, 0, 0, 0, 0, 4, 14, 31, 14, 14, 0, 0],
+    "→": [0, 0, 0, 0, 0, 4, 15, 31, 15, 4, 0, 0],
+    "↓": [0, 0, 0, 0, 0, 14, 14, 31, 14, 4, 0, 0],
+    " ": [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+}
+
+
+
+def make_bytes(ch):
+    rows = monogram[ch]
+    index = list(monogram).index(ch)  # ordered dict comes in handy
+    print('\t//', repr(ch))
+    for i, byte in enumerate(rows):
+        if not byte: continue
+        a = f'{byte:07b}'  # convert byte to zero-left-padded str
```

Comment:
I was doing some other stuff in here, reversing the order of the bits, etc. but it turned out not to be necessary.  `a` could be folded into the f-string pattern on the next line.